src/Client/Features/Rules/Fines.razor

```diff
@@ -1,6 +1,21 @@
 ﻿<MudExpansionPanel Text="Fines">
-    Owning a player that receives a fine, will result in the owner(s) receiving a fine based on the players contract value. The fines start a $1 and
-    increment by $1 dollar for each $50 in contract value with a maximum fine of $6. Fines must be called out by other owners and will be paid at the end
-    of the season. If the NFL reverses their decision, the commissioner will pay the owner(s) back the fine amount. Attempting to make a trade or waiver claim
-    that is invalid will result in a blocked claim/trade and a $5 fine.
+    <MudText Typo="Typo.h6">Player Fines</MudText>
+    Owning a player that receives a fine, will result in the owner(s) receiving a fine based on the players contract value.
+    <br>
```

Comment:
Probably better to use the built in utilities for padding/margin rather than `<br/>`.

Reply:
[Mudblazor Spacing](https://mudblazor.com/utilities/spacing#how-it-works) docs